modules/auxiliary/scanner/smb/smb_enumshares.rb

```diff
@@ -46,6 +46,8 @@ def initialize(info = {})
       [
         OptBool.new('SpiderShares', [false, 'Spider shares recursively', false]),
         OptBool.new('ShowFiles', [true, 'Show detailed information when spidering', false]),
+        OptString.new('Share', [ false, 'Show only the specified share']),
+        OptRegexp.new('HIGHLIGHT_NAME_PATTERN', [true, 'PCRE regex of resource names to highlight', 'username|password|user|pass']),
```

Comment:
```suggestion
        OptRegexp.new('HIGHLIGHT_NAME_PATTERN', [true, 'PCRE regex of resource names to highlight', 'username|password|user|pass|Groups.xml']),
```

https://vk9-sec.com/exploiting-gpp-sysvol-groups-xml/

Reply:
Ah, cool. Will get that added in. Thanks 👍